src/kcmd/grub.go

```diff
@@ -92,14 +38,28 @@ func UpdateGrub(cfg *model.InternalConfig) ([]string, error) {
 	for k, v := range params {
 		currParams[k] = v
 	}
-	grubDefault.Cmdline = model.ParamsToCmdline(currParams)
-	log.Println("Final kcmdline:", grubDefault.Cmdline)
 
-	if err := processFile(grubDefault); err != nil {
-		return nil, fmt.Errorf("error updating %s: %v", grubDefault.FilePath, err)
+	cfg.GrubCfg.Cmdline = model.ParamsToCmdline(currParams)
+	log.Println("Final kcmdline:", cfg.GrubCfg.Cmdline)
+
+	if err := processFile(cfg.GrubCfg); err != nil {
+		return nil, fmt.Errorf("error updating %s: %v",
+			cfg.GrubCfg.CustomGrubFilePath, err)
 	}
 
-	return GrubConclusion(grubDefault.FilePath), nil
+	return GrubConclusion(cfg.GrubCfg.CustomGrubFilePath), nil
+}
+
+func parseGrubCMDLineLinuxDefault(path string) (string, error) {
+	grubMap, err := ParseDefaultGrubFile(path)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse grub file: %v", err)
+	}
+	cmdline, ok := grubMap["GRUB_CMDLINE_LINUX_DEFAULT"]
+	if !ok {
+		log.Printf("WARN: GRUB_CMDLINE_LINUX_DEFAULT not found in %s", path)
```

Comment:
```suggestion
		log.Printf("GRUB_CMDLINE_LINUX_DEFAULT not found in %s", path)
```
Why should the user be warned about something that is optional?

Reply:
It's not mandatory, but by default it's there.

If isn't there, something is weird. So that's why the **WARN**.

But it could be only a note as well.

Reply:
Let's go with a normal log. We don't need to distract user's attention for this.